### DISCUSSION TITLE
Using runtime-config.json with Amplify

### DISCUSSION THREAD

**Discussion by mmuller88** [aws/aws-pdk#80] [Q&A] [answered]
The constructs here are pretty much mind-blowing. Thanks a lot! Are there references on how I can incorporate the runtime-config.json with Amplify React TS?

**Comment by cogwirrel** [comment, marked as answer]
Hey Martin!

Thanks for the feedback! 😁

I think the best example we have is part of the `awsui-react-ts-website` project here: https://github.com/aws/aws-prototyping-sdk/blob/mainline/packages/awsui-react-ts-website/samples/src/Auth.tsx#L76-L97

The runtime-config.json file is written to the root of the static website's s3 bucket at deployment time, so the website can pull it down with `fetch('/runtime-config.json')` and plug the values into Amplify :)

Hope that helps!

Cheers,
Jack

**Reply by mmuller88** [reply]
Oh boy, thank you for that gold. I never would have guessed that.

**Comment by mmuller88** [comment]
BTW your amazing work here is gaining some interest and the community can provide more feedback soon, I hope :). https://cdk-dev.slack.com/archives/C018XT6REKT/p1657088693244409

**Reply by cogwirrel** [reply]
Oh cool! I don't think I have access to that slack workspace - would be keen to take a look if you're happy and able to add me! :) My email is my github username at gmail :)

**Reply by agdimech** [reply]
Hi Martin,

Thank you for the kind words :)

+1 for adding me to the Slack channel.

**Reply by mmuller88** [reply]
@cogwirrel the slack workspace is public. Did joining work :) ?

**Reply by cogwirrel** [reply]
It didn't let me sign up at the time but I've just tried again now and I'm in! Thanks :)
